stubs/Authlib/authlib/jose/rfc7518/ec_key.pyi

```diff
@@ -0,0 +1,24 @@
+from _typeshed import Incomplete
+
+from authlib.jose.rfc7517 import AsymmetricKey
+from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKeyWithSerialization, EllipticCurvePublicKey
+
+class ECKey(AsymmetricKey):
+    kty: str
+    DSS_CURVES: Incomplete
+    CURVES_DSS: Incomplete
+    REQUIRED_JSON_FIELDS: Incomplete
+    PUBLIC_KEY_FIELDS = REQUIRED_JSON_FIELDS
+    PRIVATE_KEY_FIELDS: Incomplete
+    PUBLIC_KEY_CLS = EllipticCurvePublicKey
```

Comment:
@JelleZijlstra 

I've been playing around w/ a local checkout + stubtest to try and get this to work, but I can't seem to figure it out

Is there a known deficiency in how stubtest handles attributes set to class names? Or is it because we can't type it as that since it's not final.

Reply:
Some cases I can replace it with `type[blah] = ...` but not always, I'm not sure why.

Reply:
This does seem like a bug in stubtest. Looks like python/mypy#13316.